### DISCUSSION TITLE
Optional containers

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#408] [Ideas] [answered]
- [ ] LDAP #406 
- [ ] Mail #356 
- [ ] OSRM (Maps) #157 
- [x] Imaginary #393 
- [ ] Netdata #392 
- [ ] Fail2Ban #251 
- [x] Nextcloud Talk recording server

Any other suggestions for containers feel free to comment then I'll add them to the list.

**Comment by szaimen** [comment, marked as answer]
I fear except the Imaginary container does no of the above listed possible containers pass all requirements for integrating these new containers into AIO.
See https://github.com/nextcloud/all-in-one#requirements-for-integrating-new-containers

Imaginary will be hanlded in https://github.com/nextcloud/all-in-one/discussions/393

**Reply by szaimen** [reply]
However you might be interested in this: #1581
